src/main/java/net/minecraftforge/event/world/BlockEvent.java

```diff
@@ -444,4 +444,47 @@ public PortalSpawnEvent(IWorld world, BlockPos pos, BlockState state, NetherPort
             return size;
         }
     }
+    
+    /**
+     * Fired when when this block is right clicked by a tool to change its state.
+     * Used to determine if an axe can strip, a shovel can path, or a hoe can till.
+     * 
+     * This event is {@link Cancelable}. If canceled, this will prevent the tool
+     * from changing the block's state.
+     */
+    @Cancelable
+    public static class BlockToolInteractEvent extends BlockEvent
+    {
+
+    	private final PlayerEntity player;
+    	private final ItemStack stack;
+    	private BlockState finalState;
+    	
+		public BlockToolInteractEvent(IWorld world, BlockPos pos, BlockState originalState, PlayerEntity player, ItemStack stack) {
+			super(world, pos, originalState);
+			this.player = player;
+			this.stack = stack;
+			this.finalState = originalState;
+		}
+		
+		public PlayerEntity getPlayer()
```

Comment:
All the getter lines are wrong formatting should be same line getters.
i.e. 
```
public PlayerEntity getPlayer() { return player; }
```

Reply:
Most of the formatting for BlockEvent is incorrect then.

Reply:
I might be wrong, but that's how I've been told my triage in the past to format the getters/setters.